token/program-2022-test/tests/token_group_initialize_member.rs

```diff
@@ -0,0 +1,490 @@
+// #![cfg(feature = "test-sbf")]
```

Comment:
Was this a mistake? It looks like we do want this for `test-sbf`

Reply:
Ahh yeah, it doesn't give me IntelliSense with the feature flag on

Reply:
You should be able to configure it to enable crate features. At least, you can in vim :grin:

Reply:
I think it's purely procrastination 😭